src/game/client/components/nameplates.h

```diff
@@ -6,6 +6,7 @@
 #include <engine/shared/protocol.h>
 
 #include <game/client/component.h>
+#include <game/generated/protocol.h>
```

Comment:
```suggestion
#include <generated/protocol.h>

#include <game/client/component.h>
```

Or alternatively, this include could probably also be replaced by a single forward declaration for `struct CNetObj_PlayerInfo`.

Reply:
Did the forward declaration. I wonder when to use which.